cmd/kubeadm/app/cmd/init.go

```diff
@@ -31,15 +31,16 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/validation"
+	cmdphases "k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
```

Comment:
@dmmcquay If you have opinions, send PRs to fix, I won't stop you ;)
However, I personally don't have time to fix that given all the features I/we want to land in v1.8